osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -35,7 +40,26 @@ protected override double StrainValueAt(DifficultyHitObject current)
             currentStrain *= strainDecay(current.DeltaTime);
             currentStrain += AimEvaluator.EvaluateDifficultyOf(current, withSliders) * skillMultiplier;
 
+            if (current.BaseObject is Slider)
+            {
+                sliderStrains.Add(currentStrain);
+            }
+
             return currentStrain;
         }
+
+        public double GetDifficultSliders()
+        {
+            if (sliderStrains.Count == 0)
+                return 0;
+
+            double[] sortedStrains = sliderStrains.OrderDescending().ToArray();
+
+            double maxSliderStrain = sortedStrains.Max();
+            if (maxSliderStrain == 0)
+                return 0;
+
+            return sortedStrains.Sum(strain => 1.0 / (1.0 + Math.Exp(-(strain / maxSliderStrain * 12.0 - 6.0))));
```

Comment:
This is a wrong way of getting the amount of difficult sliders because it's gonna include sliderjumps as "difficult sliders", what is not what you want for sliderend penalty
Instead - you should use difference between strains of Aim and AimNoSliders, and check how spikey those deltas are

Reply:
In the future we can probably do something about this, but I'm not concerned as it doesn't break anything. This is still better than what exists currently.